jormungandr/src/network/mod.rs

```diff
@@ -279,6 +283,28 @@ pub fn start(
         );
     }
 
+    let peers = global_state.peers.clone();
+    let gc_err_logger = global_state.logger.clone();
+    let gc_info_logger = global_state.logger.clone();
+    global_state.spawn(
+        Interval::new_interval(Duration::from_secs(13))
```

Comment:
I'd like this to be a named const defined at the beginning of the file.

Reply:
done